.github/workflows/docs-build-update.yml

```
@@ -0,0 +1,77 @@
+
+name: Build & update docs
+
+on:
+  push:
+    branches: [ 'doc/*', 'docs/*', master, "maint/*" ]
+    tags: [ '*' ]
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
```

Need to get other branches.

```suggestion
    - uses: actions/checkout@v2
      with:
        fetch-depth: 0
```